### PR TITLE
Fix relevant notebooks

### PR DESCRIPTION
**Pull request by bvarjavand**
1. fix filepaths for uploading notebook
2. with `use_https=True` we don't need an aws key for downloading notebook
3. clean data uploading by adding to `.gitignore`

### CODE REVIEW

File: brainlit/algorithms/__init__.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import brainlit.algorithms.generate_fragments
-import brainlit.algorithms.connect_fragments
-import brainlit.algorithms.trace_analysis
-
-from brainlit.algorithms.generate_fragments import *
-from brainlit.algorithms.connect_fragments import *
-from brainlit.algorithms.trace_analysis import *
+from . import generate_fragments
```
Comment (tathey1) [inline brainlit/algorithms/__init__.py:1]: why are these imports relative?

File: brainlit/algorithms/trace_analysis/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import fit_spline, spline_fxns
```
Comment (tathey1) [inline brainlit/algorithms/trace_analysis/__init__.py:1]: why relative?

File: brainlit/utils/session.py [hunk 3]
```diff
@@ -31,6 +31,7 @@ class NeuroglancerSession:
         url: Precompued path either to a file URI or url URI. Defaults to mouselight brain1.
         mip: Resolution level to pull and push data at. Defaults to 0, the highest resolution.
         url_segments: Precomputed path to segmentation data. Optional, default None.
+        use_https: Whether to use a potentially read-only cached version of the data. Optional, default True.
```
Comment (tathey1) [inline brainlit/utils/session.py:34]: In your PR comment, you said that this addition allows for reading public aws data, but your description says something else, can you please reconcile these two descriptions?

**Review by tathey1 — CHANGES_REQUESTED** [review]
Also, i believe the reason tests fail are because you need to bump nilearn version to 0.7.0 in requirements.txt (this is a forthcoming change to the repo).

**Comment by bvarjavand** [comment]
@alishakodibagkar I'm getting this error on NeuronTrace's test
```
=========================== short test summary info ============================
ERROR brainlit/utils/tests/test_Neuron_trace.py - cloudvolume.exceptions.InfoUnavailableError: No info file was found: file:///home/circleci/project/data/test_upload_segments/info
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
```
The full log is in the most recent circleCI build for this branch. Any advice?

File: brainlit/utils/session.py (hunk 3 above)
Comment (bvarjavand) [reply brainlit/utils/session.py]: Descriptions resolved.

File: brainlit/algorithms/trace_analysis/__init__.py (hunk 2 above)
Comment (bvarjavand) [reply brainlit/algorithms/trace_analysis/__init__.py]: no reason. swapped back to absolute

File: brainlit/algorithms/__init__.py (hunk 1 above)
Comment (bvarjavand) [reply brainlit/algorithms/__init__.py]: no reason. swapped back to absolute

**Review by tathey1 — APPROVED** [review]
requested changes have been made

**Pull request merged**